src/app.rs

```diff
@@ -42,6 +43,9 @@ pub struct App {
     pub github_oauth:
         BasicClient<EndpointSet, EndpointNotSet, EndpointNotSet, EndpointNotSet, EndpointSet>,
 
+    /// Docs.rs API client
+    pub docs_rs: Arc<dyn DocsRsClient>,
```

Comment:
since we're only using this in the background job now, I guess we only need to add it to the background runner environment, right?